docs-chef-io/content/inspec/platforms.md

```diff
@@ -148,20 +132,42 @@ If you have created a `~/.azure/credentials` file as above, you may also use the
 inspec exec my-profile -t azure://2fbdbb02-df2e-11e6-bf01-fe55135034f3
 ```
 
+## AliCloud Platform Support in InSpec
+
+You will need to install AliCloud SDK version 0.8.0 and require AliCloud credentials to use the Chef InSpec AliCloud resources.
+
+### Setting up AliCloud credentials for InSpec
+
+You can configure AliCloud credentials in an [.envrc file](https://github.com/inspec/inspec-alicloud#:~:text=shell.%20(See%20example-,.envrc%20file,-)) or export them in your shell.
+
+```bash
+# Example configuration
+export ALICLOUD_ACCESS_KEY="anaccesskey"
+export ALICLOUD_SECRET_KEY="asecretkey"
+export ALICLOUD_REGION="eu-west-1"
+```
+
+### Train and InSpec Dependencies
+
+InSpec AliCloud depends on AliCloud SDK version 0.0.4, which is provided via Train AliCloud. InSpec does not ship with Train AliCloud and is explicitly listed in the Gemfile.
+
+### Destroying Created Resources Terraform
```

Comment:
Please remove this section. This is related to developer documentation for integration testing, not user documentation.

Reply:
ok

Reply:
done